packages/splitio-adapter/modules/adapter/adapter.js

```diff
@@ -50,13 +53,14 @@ const ensureUser = user => ({
   ...user,
 });
 
-const initializeUserContext = (clientSideId, user) => {
+const initializeClient = (authorizationKey, user) => {
   const factory = splitio({
     core: {
-      authorizationKey: clientSideId,
+      authorizationKey: authorizationKey,
```

Comment:
You can remove the value part. It's ES6 😉 
```
authorizationKey,
```

Reply:
True that. Nice spot will do that later.